### ISSUE TITLE
The image on the rotator banner for Free Gift is out of sync

### ISSUE THREAD

**Issue by NewmanCatalysis** [NewmanCatalysis/ITProCloudEssentials#15]
The image on the rotator banner for Free Gift is out of sync

**Comment by NewmanCatalysis** [comment]
Need specific information. Is a new image being requested?

**Comment by kengman** [comment]
Nope. the image is behind the rotation of the slide. It starts fine, but wait for them to rotate thru. The image comes after the slide does.

**Comment by javnino** [comment]
I got the same error. After the first 3 images rotate the gift comes back. The text comes first and the image appears (maybe) a second later
